deployment/refinery-modules/refinery/manifests/aws.pp

```diff
@@ -56,6 +56,13 @@
   before => Exec["solr_install"],
 }
 ->
+file { "$django_docker_engine_data_dir":
```

Comment:
`$django_docker_engine_data_dir` is a global, so should be referenced as `$::django_docker_engine_data_dir`. Also, how will this directory be created on Vagrant?